docs/snippets/setup-slack-integration.mdx

```diff
@@ -121,6 +121,7 @@ Here is the format in the yml itself:
 slack:
   token: <your_slack_token>
   channel_name: <slack_channel_to_post_at>
+  timezone: <timezone_for_timestamps>
```

Comment:
Timezone is actually agnostics to the platform it is shared on, and it is content configuration.
So I think it will be better to separate it from the slack configuration and have it as an argument on the outer level of the config yaml like:
```
slack:
  token: <your_slack_token>
  channel_name: <slack_channel_to_post_at>

timezone: <timezone_for_timestamps>
```

Reply:
In addition I think it would be nice to add a small line that explains what the use of this timezone argument 🙂